Samples/java/build.gradle

```diff
@@ -30,18 +30,18 @@ dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
 
     // Google
-    implementation "com.android.support:appcompat-v7:$rootProject.ext.supportLibVersion"
-    implementation "com.android.support:recyclerview-v7:$rootProject.ext.supportLibVersion"
-    implementation "com.android.support:design:$rootProject.ext.supportLibVersion"
-    implementation "com.android.support:cardview-v7:$rootProject.ext.supportLibVersion"
-    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
+    implementation 'androidx.appcompat:appcompat:1.0.0'
+    implementation 'androidx.recyclerview:recyclerview:1.0.0'
+    implementation 'com.google.android.material:material:1.0.0'
+    implementation 'androidx.cardview:cardview:1.0.0'
+    implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
     implementation "com.google.maps.android:android-maps-utils:$rootProject.ext.androidMapsUtilsVersion"
 
     // Phunware Dependencies
     implementation "com.phunware.location:provider-managed:$rootProject.ext.pwLocationSdkVersion"
     implementation "com.phunware.mapping:mapping:$rootProject.ext.pwMappingSdkVersion"
 
-    androidTestImplementation('com.android.support.test.espresso:espresso-core:2.2.2', {
+    androidTestImplementation('androidx.test.espresso:espresso-core:3.1.0', {
         exclude group: 'com.android.support', module: 'support-annotations'
```

Comment:
do we still need the exclude for com.android.support here?
do we need to change it to android.x?

Reply:
Good catch.  Not sure why this exclude is there, and I don't think we are even using espresso so it can probably be removed...

Reply:
Removed